Vagrantfile

```diff
@@ -156,6 +157,7 @@ end
 
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    config.vm.box_check_update = false
```

Comment:
Had this to speed up testing on my local setup. I could remove it, but I think it's better to have this if we have the constraint of checking the box versions later anyway. Would we miss any box upgrade scenario?